buildSrc/src/main/groovy/org/elasticsearch/gradle/test/StandaloneRestTestPlugin.groovy

```diff
@@ -67,6 +67,8 @@ class StandaloneRestTestPlugin implements Plugin<Project> {
         ElasticsearchJavaPlugin.configureInputNormalization(project)
         ElasticsearchJavaPlugin.configureCompile(project)
 
+        project.pluginManager.apply(RestTestBasePlugin)
```

Comment:
nit: can we group this up above with the other plugin applications?

Reply:
the original reason for this was that the plugin relies on nonInputSystemProperties. 

I've redid the general augmentation of Test tasks now and moved that into its own plugin. Another benefit of this is we removed one reused static method from ElasticsearchJavaPlugin.